ci/eval/attrpaths.nix

```
@@ -18,6 +18,7 @@
   lib ? import (path + "/lib"),
   trace ? false,
   path ? ./../..,
+  extraNixpkgsConfig ? "{}",
```

Let's make this interface 100% clear:


```suggestion
  extraNixpkgsConfigJson,
```